slimline.zsh

```diff
@@ -188,7 +188,7 @@ prompt_slimline_async_tasks() {
 
 prompt_slimline_async_init() {
   _prompt_slimline_async_pid=0
-  _prompt_slimline_async_data="${TMPPREFIX}-prompt_slimline_data"
+  _prompt_slimline_async_data="${TMPPREFIX}-${prompt_slimline_default_user}-prompt_slimline_data"
```

Comment:
It works for me...
~ ❯ ls -lah /tmp/ | grep zsh-rob
-rw-r--r--   1 rob   wheel    39B 31 Jul 14:52 zsh-rob-prompt_slimline_data

I'm using zplug to source the prompt, maybe it's the way zplug initializes the plugin that makes it work for me?